NYPLAudiobookToolkit/Core/OpenAccessSpineElement.swift

```diff
@@ -41,14 +41,14 @@ final class OpenAccessSpineElement: SpineElement {
         self.audiobookID = audiobookID
 
         guard let payload = JSON as? [String: Any],
-            let title = payload["title"] as? String,
             let urlString = payload["href"] as? String,
             let url = URL(string: urlString),
             let duration = payload["duration"] as? TimeInterval else {
                 ATLog(.error, "OpenAccessSpineElement failed to init from JSON: \n\(JSON ?? "nil")")
                 return nil
         }
-        self.title = title
+        let defaultTitleFormat = NSLocalizedString("Chapter %@", bundle: Bundle.audiobookToolkit()!, value: "Chapter %@", comment: "Default chapter title")
```

Comment:
we should add this new string to the Localized strings file.

also do we actually need the `value:` ? Just curious, I've never used that param myself. From the docs it looks like we can omit it, but I may be wrong.

Reply:
in our case value can be omitted as we don't use keys for strings. a good idea for total refactoring! )

Reply:
> we should add this new string to the Localized strings file.

we need to generate one first, there's no such a file. What languages SimplyE supports now?

Reply:
I see one in the repo root... for now it's only English (I've been maintaining the Italian one but I have not had the chance to do that here :))

Reply:
It seems to be missing in the project then. Ideally, I think, we could use keys for strings rather than remove values. In certain cases (like grammatical gender) one English string used on different screens can be two different strings in other languages. I can't give an example from the app, but that's the most common situation. (I wonder if I can do the Russian translation...)